test/com/facebook/buck/jvm/java/DefaultJavaLibraryTest.java

```diff
@@ -1382,7 +1384,9 @@ private BuildRule createJavaLibraryRule(ProjectFilesystem projectFilesystem)
           /* providedDeps */ ImmutableSortedSet.<BuildRule>of(),
           /* abiJar */ new FakeSourcePath("abi.jar"),
           /* additionalClasspathEntries */ ImmutableSet.<Path>of(),
-          options.build(),
+          options,
+          options.trackClassUsage(),
+          new JavacToJarStepFactory(options, new BootClasspathAppender()),
```

Comment:
@shs96c Correct me if I'm wrong but your TODO from line 1352 should be relatively simple to fix right? We're making c/f/b/android/BootClasspathAppender public because of this test and I don't think we need to.

@zbsz if you think it's easy to update go for it. Otherwise lets create an issue.